Account/FxAState.swift

```diff
@@ -144,7 +144,7 @@ open class FxAState: JSONLiteralConvertible {
         return JSON([
             "version": StateSchemaVersion,
             "label": self.label.rawValue,
-        ] as NSDictionary)
+        ])
```

Comment:
Reading the code to SwiftyJSON, this might trigger a different code path — they have one initializer that takes ownership of the provided object, and one that allocates a new dictionary and copies the contents. Make sure you haven't changed the behavior here.